allure-generator/src/main/javascript/translations/pl.json

```diff
@@ -147,7 +147,12 @@
       "name": "Powtórzenia"
     },
     "severity": {
-      "name": "Ważność"
+      "name": "Ważność",
+      "blocker": "blokująca",
```

Comment:
It seems to me that the Polish translation will be highly dependent on what we decide regarding the other translations.